js/src/components/contact-information/index.js

```diff
@@ -22,6 +29,50 @@ const description = __(
 const mcTitle = __( 'Enter contact information', 'google-listings-and-ads' );
 const settingsTitle = __( 'Contact information', 'google-listings-and-ads' );
 
+export function ContactInformationPreview() {
+	const phone = useGoogleMCPhoneNumber();
+
+	const handleEditClick = () => {
+		getHistory().push( getEditContactInformationUrl() );
+	};
+
+	let sectionContent;
+
+	if ( phone.loaded ) {
+		if ( phone.data.isValid ) {
+			sectionContent = (
+				<VerticalGapLayout size="overlap">
+					<PhoneNumberCard
+						isPreview
+						initEditing={ false }
+						onEditClick={ handleEditClick }
+					/>
+					<StoreAddressCard isPreview />
+				</VerticalGapLayout>
+			);
+		} else {
+			sectionContent = (
+				<NoContactInformationCard
+					onEditClick={ handleEditClick }
+					learnMoreUrl={ learnMoreUrl }
+				/>
+			);
+		}
+	} else {
+		sectionContent = (
+			<Section.Card>
+				<AppSpinner />
+			</Section.Card>
```

Comment:
❔ Would the `SpinnerCard` component be good here?

```suggestion
			<SpinnerCard />
```

Reply:
The card height will be higher and looks better. Thanks!

![image](https://user-images.githubusercontent.com/17420811/128116657-5bb499de-af8e-4146-9b4e-7c8f7d851e60.png)

Reply:
Just recalled that the `<Section.Card> <AppSpinner /> </Section.Card>` was taken for aligning the height of all loading cards on the settings page.

![Snapshot 2021-08-04 at 11 22 48](https://user-images.githubusercontent.com/17420811/128120921-8a58d93f-de31-4710-87b4-a3c9296e304d.jpg)

But `<SpinnerCard>` looks better, to make all loading UIs consistency in the Setting page, I adjusted the spinner in the DisconnectAccounts as well - https://github.com/woocommerce/google-listings-and-ads/pull/921/commits/ffba4487bb0a053dc3adb43f4b864c18478bea29